packages/connect-plugin-stellar/src/index.ts

```diff
@@ -0,0 +1,185 @@
+// eslint-disable-next-line import/no-unresolved
+import StellarSdk, { Signer, Asset, Memo, Transaction } from 'stellar-sdk';
+import BigNumber from 'bignumber.js';
```

Comment:
bignumber is missing in dependency, maybe we should not use it at all? just thinking out loud...

Reply:
listed bignumber.js in dependencies. maybe it could be removed. but still I am not sure how big numbers we might expect here.

Reply:
did some googling, looks like they have big amounts